src/backend/rest.ts

```diff
@@ -261,6 +266,10 @@ export default class RestApiBackend implements Backend {
         'This Checkmk data source is only compatible with Checkmk Cloud and Checkmk MSP, but you are trying to connect to another edition.'
       );
     }
+    // CSE is never supported
```

Comment:
in the signed version of the grafana plugin this will never be executed, as the previous check will fails first. should we reoder this to get a more specifc error message?